src/commands/pull.ts

```diff
@@ -0,0 +1,26 @@
+import {
+    checkIfOnline,
+    getProjectSettings,
+    LOG,
+    spinner,
+ } from './../utils';
+
+import {
+    fetchProject,
+    writeProjectFiles,
+} from './../files';
+
+/**
+ * Force downloads all Apps Script project files into the local filesystem.
+ * @param cmd.versionNumber {number} The version number of the project to retrieve.
+ *                             If not provided, the project's HEAD version is returned.
+ */
+export const pull = async (cmd: { versionNumber: number }) => {
```

Comment:
Maybe export default if we are going with 1 command per file?

```ts
export default async (cmd: { versionNumber: number }) => {
```

Reply:
Great idea